spec/PeriodicBackgroundSync-index.bs

```diff
@@ -151,3 +151,9 @@ The <code><dfn method for=PeriodicSyncManager title="register(tag, options)">reg
             1. Set |currentRegistration|'s associated [=options=] to |options|.
             1. Calculate an appropriate <var>timeToFire</var> for |currentRegistration| based on the [=minInterval=] value of |options|, and any other considerations the user agent has implemented per recommendations in [[#privacy]] and Resource Usage. The time interval between now and |timeToFire| MUST be greater than or equal to the [=minInterval=] specified in |options|. Set the |currentRegistration|'s [=time to fire=] to |timeToFire|.
     1. [=Resolve=] |promise|.
+
+The <code><dfn method for=PeriodicSyncManager title="getTags()">getTags()</dfn></code> method when invoked, MUST return [=a new promise=] <var>promise</var> and run the following steps [=in parallel=]:
+1. Let <var>serviceWorkerRegistration</var> be the {{PeriodicSyncManager}}'s associated [=service worker registration=].
+1. Let <var>currentTags</var> be a new [=sequence=].
```

Comment:
Wouldn't we want to use `list` here?

Reply:
Did you mean any of these lists:
spec:url; type:dfn; text:list
spec:infra; type:dfn; text:list
Or something else? I've left it unlinked for now.

Reply:
Infra's list.

Reply:
done.